packages/components/src/components/RunMetadataColumn/RunMetadataColumn.jsx

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2025 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { usePrefix } from '@carbon/react';
+
+const RunMetadataColumn = ({ columnHeader, columnContent }) => {
+  if (!columnContent) {
+    return null;
+  }
+  const carbonPrefix = usePrefix();
+  return (
+    <div className="tkn--metadata-column">
+      <h3 className={`${carbonPrefix}--label tkn--metadata-column-header`}>
+        {columnHeader}
+      </h3>
+      <div className="tkn-metadata-column-content">{columnContent}</div>
```

Comment:
classname should use `tkn--` prefix


```suggestion
      <div className="tkn--metadata-column-content">{columnContent}</div>
```